src/pubkey.h

```diff
@@ -205,4 +210,13 @@ struct CExtPubKey {
     bool Derive(CExtPubKey& out, unsigned int nChild) const;
 };
 
+/** Users of this module must hold an ECCVerifyHandle. The constructor and
+ *  destructor of these are not allowed to run in parallel, though. */
```

Comment:
Done. Only for the refcount though, as the context variable is needed outside of the class (for obvious reasons).